plugins/block-backed-procedures/test/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Block test.
+ */
+
+import * as Blockly from 'blockly';
+import {createPlayground} from '@blockly/dev-tools';
+import '../src/index';
+
+
+// TODO: Edit list of blocks.
+const allBlocks = ['block_template'];
+
+/**
+ * Create a workspace.
+ * @param {HTMLElement} blocklyDiv The blockly container div.
+ * @param {!Blockly.BlocklyOptions} options The Blockly options.
+ * @return {!Blockly.WorkspaceSvg} The created workspace.
+ */
+function createWorkspace(blocklyDiv, options) {
+  const workspace = Blockly.inject(blocklyDiv, options);
+  return workspace;
+}
+
+document.addEventListener('DOMContentLoaded', function() {
+  const defaultOptions = {
+    toolbox: `<xml xmlns="https://developers.google.com/blockly/xml">
```

Comment:
Is this part of the template? We should probably update this to be JSON instead.

Reply:
It is indeed part of the template, I can do that rq.